RFCs/FS-1135-random-functions-for-collections.md

```diff
@@ -179,28 +178,21 @@ Please list the reasonable expectations for tooling for this feature, including
 * Colorization
 * Brace/parenthesis matching
 
-**Should work just like for other collections functions**
+**These will work just like for other collections functions**
 
 ## Performance
 
-Please list any notable concerns for impact on the performance of compilation and/or generated code
-
-* For existing code **Existing code should not be affected**
-* For the new features **Performance should be respected when implementing this feature, since it can be used in performance-sensitive scenaris**
+* For existing code **Existing code won't be affected**
+* For the new features **Performance should be respected when implementing this feature, since it can be used in performance-sensitive scenarios**
 
 ## Scaling
 
-Algorithmic complexity of the new features should be O(n) or less. 
+Algorithmic complexity of the new features should be O(n) for list and seq functions. O(1) for most array functions (except shuffle).
 
 ## Culture-aware formatting/parsing
 
 N/A
 
 # Unresolved questions
 
-It's unclear, if more sophisticated overloads should be added:
- - Weights parameter for randomChoices function
- - Counts parameter for randomSample function
-
-~~In .NET 6 and higher [Random.Shared](https://learn.microsoft.com/en-us/dotnet/api/system.random.shared) is available, but as soon as F# Core only targets standard, it can't use it. Is targeting higher .NET possible (not just for this feature, maybe some others need it)?
-Same question about new [.NET 8 apis](https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8#methods-for-working-with-randomness), they could be reused in theory~~
\ No newline at end of file
+[Additional function suggested](https://github.com/fsharp/fslang-design/discussions/731#discussioncomment-9328918)
```

Comment:
Thanks for linking the open point here. Let's resolve that soon(ish), so we can replace this simply with nothing, and I can merge this proposal for you, so we can start implementing it before the deadline of the next F# gets too close.

```suggestion
None
```